File: app/src/main/java/com/ipd/taxiu/ui/activity/coupon/ExchangeRecordActivity.java
package com.ipd.taxiu.ui.activity.coupon;

import android.os.Bundle;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;

import com.ipd.taxiu.R;
import com.ipd.taxiu.adapter.ExchangeRecordAdapter;
import com.ipd.taxiu.ui.BaseUIActivity;
import com.ipd.taxiu.ui.fragment.coupon.ExchangeRecordFragment;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Miss on 2018/7/30
 * 兑换记录
 */
public class ExchangeRecordActivity extends BaseUIActivity {

    @Override
    protected int getContentLayout() {
        return R.layout.activity_container;
    }

    @Override
    protected void initView(@Nullable Bundle bundle) {
        initToolbar();
    }

    @Override
    protected void loadData() {
       getSupportFragmentManager().beginTransaction().replace(R.id.fl_container, ExchangeRecordFragment.newInstance()).commit();
    }


    @Override
    protected void initListener() {

    }

    @NotNull
    @Override
    protected String getToolbarTitle() {
        return "兑换记录";
    }
}

File: app/src/main/java/com/ipd/taxiu/ui/activity/coupon/ExchangeRecordActivity.kt
package com.ipd.taxiu.ui.activity.coupon

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.ipd.taxiu.R
import com.ipd.taxiu.ui.BaseUIActivity
import com.ipd.taxiu.ui.fragment.coupon.ExchangeRecordFragment

/**
Created by Miss on 2018/8/10
兑换记录
 */
class ExchangeRecordActivity:BaseUIActivity() {
    companion object {
        fun launch(activity:Activity){
            val intent = Intent(activity,ExchangeRecordActivity::class.java)
            activity.startActivity(intent)
        }
    }
    override fun getContentLayout(): Int = R.layout.activity_container
    override fun getToolbarTitle(): String ="兑换记录"

    override fun initView(bundle: Bundle?) {
        initToolbar()
    }

    override fun loadData() {
        supportFragmentManager.beginTransaction().replace(R.id.fl_container,ExchangeRecordFragment.newInstance()).commit()
    }

    override fun initListener() {
    }
}